Package.swift

```diff
@@ -91,7 +99,30 @@ let package = Package(
         .product(name: "SwiftSyntax", package: "swift-syntax"),
       ]
     ),
-
+    .plugin(
+      name: "Format Source Code",
+      capability: .command(
+        intent: .sourceCodeFormatting(),
+        permissions: [
+          .writeToPackageDirectory(reason: "This command formats the Swift source files")
+        ]
+      ),
+      dependencies: [
+        .target(name: "swift-format")
+      ]
+    ),
+    .plugin(
+      name: "Lint Source Code",
+      capability: .command(
+        intent: .sourceCodeFormatting(),
```

Comment:
What's the behavior of `swift package format-source-code` when there are two plugins with this intent?

I wonder if we should use something like `.custom(verb: "lint-source-code")` here instead.

Reply:
I fixed it. https://github.com/zunda-pixel/swift-format/commit/8bcb0bb94af287b080aa5d631360cc24737d9ba0

```shell
swift-format$ swift package plugin --list
‘format-source-code’ (plugin ‘Format Source Code’ in package ‘swift-format’)
‘generate-manual’ (plugin ‘Generate Manual’ in package ‘swift-argument-parser’)
‘lint-source-code’ (plugin ‘Lint Source Code’ in package ‘swift-format’)
```